spec: section: source-vocabulary: drop non-chartered and RML-Core reference formulations (#98)
kg-construct/rml-io: spec/section/source-vocabulary.md

such as in the example above, by using the `rml:SQL2008Table`
as `rml:referenceFormulation`.

More examples with corresponding reference formulations and access descriptions

> can be found in the [[RML-IO-Registry]] which are optionally to implement by

Reviewer: ```suggestion
can be found in the [[RML-IO-Registry]] which MAY be implemented by
```

> implementations while the ones specified in this specifications are required to

Reviewer: ```suggestion
implementations while the ones specified in this specifications MUST
```
